src/style.css

```diff
@@ -1,3 +1,63 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
+
+
+body {
+  font-family: Roboto, sans-serif;
```

Comment:
It's better to set font-family and default font styles to html selector

Reply:
I moved font-family from body to html selector.